express/mvc/controller/PayhubController.js

```diff
@@ -508,11 +407,21 @@ class PayhubController {
         res.status(200).json(result);
       })
       .catch(error => {
-        if (error.statusCode) {
-          res.status(400).send(error);
-        }
+        return this.handleError(res, error);
       });
   }
+
+  handleError(res, error){
+    let msg = "";
+    if (error.message){
+      msg = error.message;
```

Comment:
- if (myVariable) can also throw an error in two cases. The first is when the variable hasn't been defined which throws a ReferenceError.

Reply:
could you add cause too ? As mentioned node is not type safe error could have any format, :disappointed: sometimes node use error.message or internal errors uses error. cause
[14:43](https://hmcts-reform.slack.com/archives/D05GP3K5HT8/p1718026988359639)
2024-06-07T10:32:26+01:00 - info: [PAYBUBBLE: security.js -> error] {"error":"Error: getUserDetails() call failed","cause":"Error: getaddrinfo ENOTFOUND idam-api.demo.platform.hmcts.net"}